RELEASE.md

```diff
@@ -0,0 +1,7 @@
+Release type: patch
```

Comment:
shall we mark this as minor? 😊

Reply:
Hrmmm, although this is a fix the refactor is large enough to be considered a minor. Will change it :)